boat-maven-plugin/src/main/templates/htmlDocs2/index.mustache

```diff
@@ -232,74 +231,71 @@
-                          </div>
-                          <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-java">
-                            <pre class="prettyprint"><code class="language-java">{{>sample_java}}</code></pre>
-                          </div>
-
-                          <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-android">
-                            <pre class="prettyprint"><code class="language-java">{{>sample_android}}</code></pre>
-                          </div>
-  <!--
-  <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-groovy">
-  <pre class="prettyprint language-json prettyprinted" data-type="json"><code>Coming Soon!</code></pre>
-  </div> -->
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-objc">
-                              <pre class="prettyprint"><code class="language-cpp">{{>sample_objc}}</code></pre>
-                            </div>
-
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-javascript">
-                              <pre class="prettyprint"><code class="language-js">{{>sample_js}}</code></pre>
-                            </div>
-
-                            <!--<div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-angular">
-              <pre class="prettyprint language-json prettyprinted" data-type="json"><code>Coming Soon!</code></pre>
-            </div>-->
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-csharp">
-                              <pre class="prettyprint"><code class="language-cs">{{>sample_csharp}}</code></pre>
-                            </div>
-
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-php">
-                              <pre class="prettyprint"><code class="language-php">{{>sample_php}}</code></pre>
-                            </div>
-
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-perl">
-                              <pre class="prettyprint"><code class="language-perl">{{>sample_perl}}</code></pre>
-                            </div>
-
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-python">
-                              <pre class="prettyprint"><code class="language-python">{{>sample_python}}</code></pre>
-                            </div>
-
-                            <div class="tab-pane" id="examples-{{baseName}}-{{nickname}}-0-rust">
-                              <pre class="prettyprint"><code class="language-rust">{{>sample_rust}}</code></pre>
-                            </div>
-                          </div>
+<!--                        <p>-->
```

Comment:
Why not just delete these lines? Commented out code is confusing; you can always bring it back - we use it.

Reply:
I agree Jasper. I've deleted it now